src/App.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Buttons } from './components/Buttons';
+import { GoodsList } from './components/GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export class App extends PureComponent {
+  state = {
+    visible: [],
+  }
 
-export default App;
+  handleGetAll = async() => {
+    await getAll()
+      .then((goods) => {
+        this.setState({
+          visible: goods,
+        });
+      });
+  }
```

Comment:
Некорректное использование **async await**.

```suggestion
  handleGetAll = async () => {
    const goods = await getAll();
    
    this.setState({ visible: goods });
  }
```